stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,1004 @@
+                                    <i class="sap-icon--resize"></i>
+                                </button>
+                                <button class="fd-button fd-button--transparent" aria-label="Exit fullscreen">
+                                    <i class="sap-icon--exitfullscreen"></i>
+                                </button>
+                                <button class="fd-button fd-button--transparent" aria-label="Close">
+                                    <i class="sap-icon--decline"></i>
+                                </button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="fd-dynamic-page__subtitle"> Oversized multimaterial sneakers with quilted effect </div>
+            </div>
+            <div class="fd-dynamic-page__collapsible-header fd-dynamic-page__collapsible-header--xl" role="region" id="fddplhvc3" aria-hidden="false"  aria-label="Pinned header">
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ullam possimus corrupti architecto perspiciatis, 
+                delectus necessitatibus incidunt numquam asperiores tenetur iure. 
+                Cum consequuntur impedit repellendus esse, facere autem optio consequatur nobis?
+            </div>
+            <div class="fd-dynamic-page__collapsible-header-visibility-container">
+                <div class="fd-dynamic-page__collapsible-header-visibility-container--left-gradient"></div>
+                <div role="toolbar" aria-label="Header actions" class="fd-dynamic-page__collapsible-header-visibility-container--button-group">
+                    <button class="fd-dynamic-page__collapse-button fd-button fd-button--compact" 
+                            aria-label="See More" 
+                            aria-expanded="true" 
+                            aria-haspopup="true" 
+                            aria-controls="fddplhvc3">
+                            <i class="sap-icon--slim-arrow-up"></i>
+                    </button>
+                </div>
+                <div class="fd-dynamic-page__collapsible-header-visibility-container--right-gradient"></div>
+            </div>
+        </header>
+        <div class="fd-dynamic-page__content fd-dynamic-page__content--xl">
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam 
+            eum tempore aliquid dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam eum tempore aliquid 
+            dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+        </div>
+    </section>
+`;
+
+onlyCollapse.storyName = 'Only Expand/Collapse';
```

Comment:
The dynamic page can be displayed without a pin button, giving users the option to only expand or collapse the header area.